services/api-gateway/src/simcore_service_api_gateway/endpoints_user.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, Depends, Security
+
+from .auth import get_current_active_user, get_current_user
+from .schemas import User
+
+router = APIRouter()
+
+
+@router.get("/users/me/", response_model=User)
+async def read_users_me(current_user: User = Depends(get_current_active_user)):
+    return current_user
+
+
+@router.get("/users/me/projects/")
+async def list_own_projects(
+    current_user: User = Security(get_current_active_user, scopes=["projects"])
+):
+    return [{"project_id": "Foo", "owner": current_user.username}]
```

Comment:
I think I would need some introduction to this thing... it looks nice the way you parametrize things

Reply:
I will be happy to go in detail with you on this.